README.md

```diff
@@ -1,67 +1,58 @@
 # Stylix
 
-Stylix is a NixOS module which applies the same colour scheme, font and
-wallpaper to a range of applications and desktop environments.
+## About
 
-## What's this?
+Stylix is a theming framework for [NixOS](https://nixos.org), [Home
+Manager](https://github.com/nix-community/home-manager), and
+[nix-darwin](https://github.com/LnL7/nix-darwin) that applies color schemes,
+wallpapers, and fonts to a wide range of applications.
 
-[base16.nix](https://github.com/SenchoPens/base16.nix#readme) allows you to
-import colours from [base16](https://github.com/chriskempson/base16#readme)
-into Nix code. Stylix takes this a step further:
-
-- Automatically colours and changes the font of apps
-- Sets your wallpaper
-- Exports the colour scheme to be used manually for anything we missed
-- Can also generate themes based on an image
-
-For those not familiar with [NixOS](https://nixos.org/) and
-[Home Manager](https://github.com/nix-community/home-manager#readme):
-
-- NixOS is a Linux distribution
-- Home Manager is a program which runs anywhere
-- Both use the Nix language and package manager
-- Both let you install programs and change settings via code
-
-Stylix supports either NixOS + Home Manager, or Home Manager on its own.
-Certain features are only available with NixOS.
+Unlike color scheme providers such as
+[base16.nix](https://github.com/SenchoPens/base16.nix) or
+[nix-colors](https://github.com/Misterio77/nix-colors), Stylix goes further by
+applying themes to supported applications, following the "it just works"
+philosophy.
 
 ## Resources
 
-Please refer to the [Stylix book](https://danth.github.io/stylix/)
-for instructions and a list of supported apps.
-
-For a visual guide, watch the [*Ricing Linux Has Never Been Easier | NixOS +
-Stylix*](https://youtu.be/ljHkWgBaQWU) YouTube video by
-[Vimjoyer](https://www.youtube.com/@vimjoyer).
-
-> [!NOTE]
->
-> It's now necessary to include `stylix.enable = true` in your configuration
-> for any other settings to take effect. This is not mentioned in the video
-> linked above.
-
-If you have any questions, you are welcome to
-join our [Matrix room](https://matrix.to/#/#stylix:danth.me),
-or ask on [GitHub Discussions](https://github.com/danth/stylix/discussions).
```

Comment:
We should keep the discussions link for those who don't use Matrix, and to encourage using discussions rather than issues where appropriate.

(This text is also rendered at [stylix.danth.me](https://stylix.danth.me), so linking back to GitHub is worthwhile.)

Reply:
Yes. I accidentally removed it at some point.